### DISCUSSION TITLE
CTRL+Z to restore original "save as" file name

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#139] [Ideas]
Accidentally pressing a key after the "save as" dialogue or clicking on an existing file appears causes the pre-filled file name to be lost. The dialogue needs to be opened again to get back the pre-filled file name. For example, a file needs to be downloaded again.

Please add support for CTRL+Z to restore the original file name or a button for "restore original file name".

**Comment by JosephMcc** [comment]
The file save and open dialogs are provided by the toolkit the application is written in. Mint does not develop nor have control over these.
